### DISCUSSION TITLE
추가정보 입력을 위한 '나중에 하기' 기능

### DISCUSSION THREAD

**Discussion by clyde-yoon** [team-final/albaform#122] [Ideas]
++ 초창기에 얘기 나눴던 주제에 대해서 확고히 굳혀두고싶은게 있는데
사장님 회원가입 시 
이메일 / 패스워드 / 네임 / 닉네임 / role / 가게이름 / 가게번호 / 사장번호 / 위치
가 리퀘스트 바디인데,

여기서 추가적인 정보
입력을 나중에 하기 로 하냐 안하냐
정도 궁금합니다.
(다음 아래의 3가지 전제조건은 role이 owner일때의 경우입니다)

이메일 / 닉네임 / 비밀번호 -> 만 입력해도 회원가입 가능.
이메일 / 비밀번호 -> 만 있으면 회원가입 불가능 (500상태코드 인터넷 서버 에러)
이메일 / 네임 / 비밀번호 -> 만 입력하면 회원가입 불가능 (500상태코드 인터넷 서버 에러)

따라서 step-1 에서만 회원가입을하고 추가적인 정보입력을 위한 사람을 위해 
닉네임에 Owner(5자리 난수) 를 입력해서 만들어버리는 기능 을 추가할까요 말까요

추후 추가적인 정보관련해선 로직이 좀 더 필요해질 수 있는 상태이긴 합니다.

알바폼 만들기를 클릭했을 떄 로직 한개가 추가되어야합니다.

찬반 투표로 빠르게 정해봅시다!!!!

**Comment by JuhyeokC** [comment]
관련 API `request body` 요구사항 입니다
```json
{
  "email": "example@email.com",
  "password": "password",
  "name": "string",
  "nickname": "string",
  "role": "OWNER | APPLICANT",
  "storeName": "string | undefined",
  "storePhoneNumber": "string | undefined",
  "phoneNumber": "string | undefined",
  "location": "string | undefined"
}
```
![image](https://github.com/user-attachments/assets/c88a3aeb-b0c1-49f9-8671-4e85a037d227)

**Comment by JiinSeok** [comment]
나중에 하기 대찬성입니다!
지원하기/폼만들기 시점에 검사해서 nickname 없으면 정보입력 페이지로 리다이렉트 하면 된다고 생각해요!
